datastore/src/main/kotlin/com/openlattice/collaborations/controllers/CollaborationsController.kt

```diff
@@ -41,14 +43,24 @@ class CollaborationsController : AuthorizingComponent, CollaborationsApi {
 
     @Timed
     @GetMapping(
-        value = ["", "/"],
+        value = [ALL_PATH],
         produces = [MediaType.APPLICATION_JSON_VALUE]
     )
-    override fun getCollaborations(): Iterable<Collaboration> {
+    override fun getAllCollaborations(): Iterable<Collaboration> {
         val authorizedCollaborationIds = getAllAuthorizedCollaborationIds()
         return collaborationService.getCollaborations(authorizedCollaborationIds).values
     }
 
+    @Timed
+    @GetMapping(
+            value = ["", "/"],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getCollaborations(@RequestParam( ID_PARAM ) ids: Set<UUID>): Map<UUID, Collaboration> {
```

Comment:
Do you want this in as the body or in the URL? That URL will get long real fast.

Reply:
```suggestion
    override fun getCollaborations(@RequestParam( IDS_PARAM ) ids: Set<UUID>): Map<UUID, Collaboration> {
```

Reply:
using GET with a body, or implementing POST with body with the intention of using it as a GET both deviate from what is expect from REST.

length of url isn't really a concern. if we really run into character limitations, it should be handled by limiting API calls to a certain number of ids per request.

Reply:
The comment about GET makes sense, though I am not as devoted to REST philosophy when (ab)using it for RPC. You'll have to deal with paging eventually so I think this is fine for now.

Reply:
Also there is a trade off here for debugging/testing this API, but I know where those tickets will go :)